src/it/hosting-maven_release_plugin_version-ok/invoker.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
So @stephenc copy-pasted them from another Maven plugin, headers and all. No need to preserve the ASF license info, since, well, this isn't ASF.